doc/developers-guide/accumulators.md

```diff
@@ -1,9 +1,19 @@
 # Accumulators
 
-XMPP stanza processing starts in the `ejabberd_c2s` module, which receives the stanza from a socket, or in `ejabberd_s2s_in` which receives stanzas from other MIM clusters.
+XMPP stanza processing starts in the `ejabberd_c2s` module, which receives the stanza from a socket, or in `ejabberd_s2s_in` which receives stanzas from federated XMPP clusters.
 The stanza is processed and eventually it and/or other messages are sent out, either to the original sender, to another c2s process within the same MongooseIM installation, or to another XMPP server.
 
-At the beginning of the main processing chain an accumulator is created containing the stanza and other values.
+At the beginning of the main processing chain an accumulator is created containing the original stanza, some values extracted from stanza and additional info:
```

Comment:
At the beginning of the main processing chain an accumulator is created containing the original stanza, some values extracted from the stanza and additional info:

or 

At the beginning of the main processing chain an accumulator is created containing the original stanza, some values extracted from it and additional info: